src/identity-handler.js

```diff
@@ -19,30 +18,38 @@ identityApiRequest has the schema:
 For more userIdentity types, see http://docs.mparticle.com/developers/sdk/javascript/identity#allowed-identity-types
 */
 
-var identityHandler = {
-    onIdentifyCompleted: function(mParticleUser, identityApiRequest) {
-
-    },
-    onLoginCompleted: function(mParticleUser, identityApiRequest) {
-
-    },
-    onLogoutCompleted: function(mParticleUser, identityApiRequest) {
-
-    },
-    onModifyCompleted: function(mParticleUser, identityApiRequest) {
-
-    },
-    onUserIdentified: function(mParticleUser, identityApiRequest) {
-
-    },
+function IdentityHandler(common) {
+    this.common = common || {};
+}
+IdentityHandler.prototype.onIdentifyCompleted = function(
```

Comment:
all but onUserIdentified should be present tense, a la 
https://github.com/mparticle-integrations/mparticle-javascript-integration-google-tag-manager/blob/master/src/identity-handler.js

Reply:
Good catch. I have dishonored myself by missing this. 😞